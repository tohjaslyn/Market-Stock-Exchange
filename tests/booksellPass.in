add trader1 100
add trader2 100
import trader2 ABC 10
sell trader2 ABC 5 2
sell trader2 ABC 5 2.50
book sell
exit
