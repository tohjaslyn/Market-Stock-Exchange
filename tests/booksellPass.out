$ Success.
$ Success.
$ Trader now has 10.00 units of ABC.
$ No trades could be made, order added to sell book.
$ No trades could be made, order added to sell book.
$ 0001: SELL 5.00xABC @ $2.00
0002: SELL 5.00xABC @ $2.50
$ Have a nice day.
